src/ttsim/unit_converters.py

```diff
@@ -0,0 +1,976 @@
+from __future__ import annotations
+
+from typing import overload
+
+TIME_UNIT_LABELS = {
+    "y": "Year",
+    "q": "Quarter",
+    "m": "Month",
+    "w": "Week",
+    "d": "Day",
+}
+
+_Q_PER_Y = 4
+_M_PER_Y = 12
+_W_PER_Y = 365.25 / 7
+_D_PER_Y = 365.25
+
+
+@overload
+def y_to_q(value: int) -> int: ...
+
+
+@overload
+def y_to_q(value: float) -> float: ...
+
+
+def y_to_q(value: float) -> int | float:
+    """
+    Converts values on year level to values on quarter level.
```

Comment:
```suggestion
    Convert values (stocks) measured on the yearly level to values on the quarterly level.
```
Now that we expose it, let's be picky about the docstrings.

Also, get rid of the Parameters/Returns section. Does not add any information any more.